### ISSUE TITLE
Mount Request Mega Thread

### ISSUE THREAD

**Issue by fatyg5** [ObliqueNET/Stratos#153]
Use this thread to request to make a Pokémon rideable. Use the format below to request. Do not post things like "WE SHOULD MAKE RATTATA FLYABLE BECAUSE", if you want a mount, give me a genuine reason. No wrong answers, but I'm not going to let you ride a Joltik, which would crush it. An example of a correct post will also be below.

Pokémon Name:

What type of mount:

Why we should make this Pokémon rideable:

**Comment by fatyg5** [comment]
Pokémon Name: Kyogre

What type of mount: Water

Why we should make this Pokémon rideable: Kyogre is a legendary fish Pokémon and should be able to swim with me on top.

**Comment by ChildishGambyno** [comment]
Pokémon Name: 
Dawn Wings Necorzma/ Dusk Mane Necrozma
Type of mount: 
Flying/Land
Why?: 
Both Solgaleo and Lunala are rideable so I feel it would be right to make thier fusion with necrozma rideable as well.

**Comment by Vodschou** [comment]
Pokémon Name: abomasnow (mega?)

What type of mount: Land rideable

Why we should make this Pokémon rideable: Because in its mega it hunches perfectly forward and it is a giant to ride

**Comment by darknegadude** [comment]
Pokémon Name: Dragalge

What type of mount: Water

Why we should make this Pokémon rideable: both it and Kingdra are based on similar real world animals although only Kingdra is rideable, as well as Dragalge's normal sized model looking like it would be rideable

**Comment by Oshirisu** [comment]
Pokémon Name:
The Tapus (Tapu Lele, Bulu, Fini, Koko)
What type of mount:
Flying
Why we should make this Pokémon rideable:
In the pokemon games in multiple occasions we see them flying, and also they look like they could be rideable pokemon.

**Comment by Ferruccio88** [comment]
Pokémon Name:
The Eeveelutions
What type of mount:
Land (Water for Vaporeon possibly)
Why we should make this Pokémon rideable
ya their are a lot of rideable dog/fox kind of like animal Pokémon such as absol ninetales ect but i just think it would be cool to beable to ride the eeveelutions as they are some of the most well known Pokémon out their and they are about the rideable size as they are around the same size as a lot of rideable land Pokémon so i just think it would be cool to see them rideable

**Comment by Tiny-tender-hearted-Digit** [comment]
Pokémon Name: Decidueye

What type of mount: Flying

Why we should make this Pokémon rideable:
It's a bird, and it has wings. It looks big enough to ride, (there are mons you can ride that are the same size if not smaller.) and plus, it can apparently fly in the game similar to other mons like trumbeak. (Which are often seen flying around terra spawn.) And yet, despite all this, it's not a flying mount? It really looks like you could just climb right onto it's back and fly off into the sunset with it. Perhaps you could make that a reality?

**Comment by Tiny-tender-hearted-Digit** [comment]
Pokémon Name: Pelipper

What type of mount: Water

Why we should make this Pokémon rideable:
I went onto singleplayer in creative mode and wanted to see if you could ride pelipper, and apparently, you could! But what I found odd was that despite being a water type, pelipper was only a flying mount, and I fell off when I tried to go underwater? Pelipper should be a water and flying mount.

**Comment by Vinnydudes** [comment]
Pokémon Name: Genesect

What type of mount: Flying 

Why we should make this Pokémon rideable: In the anime, it is able to fly around in its “high-speed flight” form, similar to a hoverboard. In the Genesect movie Ash also rode one in the flying form.

**Comment by lolanonymous** [comment]
> Pokémon Name: Pelipper
> 
> What type of mount: Water
> 
> Why we should make this Pokémon rideable:
> I went onto singleplayer in creative mode and wanted to see if you could ride pelipper, and apparently, you could! But what I found odd was that despite being a water type, pelipper was only a flying mount, and I fell off when I tried to go underwater? Pelipper should be a water and flying mount.

we can't do both water and flying mounts

**Comment by HydroZure** [comment]
Pokémon Name: Mightyena

What type of mount: Land

Why we should make this Pokémon rideable: 
Mightyena would like a good pokemon to ride on and looks like it feels comfortable riding mightyena.

**Comment by HydroZure** [comment]
Pokémon Name: Persian 

What type of mount: Land

Why we should make this Pokémon rideable:
Persian can be ridden in pokemon Let's go pikachu and eevee, and it will be fast.

**Comment by ZexyKun** [comment]
I'm just suggesting some edits to mounting Giratina (Origin forme) and Rayquaza (Mega Evolved)

Pokémon Name: Giratina (Origin)

What type of mount: Flying

Why we should make this Pokémon rideable: When riding Giratina in his origin form, it's impossible to see. His body constantly moves in and out of your FOV. Is there some way to make it so that the character is closer to his head to see over it? 

Pokémon Name: Rayquaza

What type of mount: Flying

Why we should make this Pokémon rideable: A sort of similar issue to the one with Giratina. In his normal form, the player is put on Rayquaza's head and vision is superb. When mega-evolved, it puts the player on his back, and while not obstructing vision the same way Giratina does, it's still pretty hard to see where I'm going. Can we adjust it so the player is closer to the head to see over it?

**Comment by Tiny-tender-hearted-Digit** [comment]
> 
> 
> > Pokémon Name: Pelipper
> > What type of mount: Water
> > Why we should make this Pokémon rideable:
> > I went onto singleplayer in creative mode and wanted to see if you could ride pelipper, and apparently, you could! But what I found odd was that despite being a water type, pelipper was only a flying mount, and I fell off when I tried to go underwater? Pelipper should be a water and flying mount.
> 
> we can't do both water and flying mounts

Yes you can, Swanna is both a flying and water mount, same with lugia. I use swanna all the time because it's so convenient for riding, and I tested lugia on singleplayer, so I would know.

**Comment by nugsy** [comment]
Pokémon Name: Chandelure

What type of mount: Flying

Why we should make this Pokémon rideable: swinging from a ghost chandelier through the skies is cool

**Comment by jv492** [comment]
Pokémon Name: Volcarona

What type of mount: Flying

Why we should make this Pokémon rideable: This a flying pokemon that you could ride in the games. I think we great to have in the game.

**Comment by mairai** [comment]
Pokémon Name: Necrozma

What type of mount: Flying

Why we should make this Pokémon rideable: I want him to hold me in his giant hands. Please. Let me have this. He's capable of it ;w;

**Comment by mairai** [comment]
Pokemon name: Sharpedo

Type of mount: water

Why: you can do it in the games, so

**Comment by doughnutchicken** [comment]
In the interest of speed, Ras' has allowed me to write this without reasons for each suggestion, they will be reviewed when added.

eeveeloutions

exeggutor/alolan

abomasnow/mega

sharpedo

necrozma all forms

volcarona

chandelure

houndoom

garbador

shaymin skyform

kangaskan ride in pouch if possible

deoxys and forms


this is what the people wanted

**Comment by JellyBlade** [comment]
Pokemon Name: Araquanid

What type of mount: Land and/or water mount, water makes the most sense, but it could also be a land mount just as well.

Why we should make this Pokémon rideable: It's a pretty large pokemon, and spiders are stupid strong in real life comparatively, so the weight of its rider wouldn't be too much for it. 

And for a personal anecdote, when I first saw my friend's araquanid in-game, my first thought was "yes, but can I ride it?", which I was sorely disappointed when I could not.
